### PR TITLE
Stefan Clem - Sprint Challenge Testing

### PR DESCRIPTION
**Pull request by iepoch**

### CODE REVIEW

File: Answers.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+> The differences between descibe and it is that Describe will be the entire Test Suite for test cases to live in. The it referes to each seperate test case that you create for that specific test suite.
+1. What is the point of `Test Driven Development`? What do you think about this approach?
+> The point of the Test Driven Development is to understand how the product works before codeing it. If you understand and know the corner cases of your product then you can develop more cleanly makeing sure it aglins with the test you created. In my honest opion Test Driven Development can be bennifical but it may not work for all cases. There are specific jobs just for QA personel who do Automated test. Developers in Agile work enviorments have to deploy fast. And somtimes dont have enough time to spend writing the test vs writing the code. Hence the need for QA personel to write the test to make sure it does what it suppose to do. So while this is great for smaller pieces of code 'Unit Test' a whole integration and regression this would not be idea.
```
Comment (juliankohlman) [inline Answers.md:4]: Great breakdown of TDD Stefan 👍 You give a good explanation and blend it well with your thoughts on the techniques.

File: api/server.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+
+const server = express();
+
+const games = require('../arcade/arcade')
+
+server.use(express.json());
+
+
```
Comment (juliankohlman) [inline api/server.js:9]: Your endpoints are thorough Stefan 👍 It's good to see you using past patterns/techniques like database helpers in your code.

File: api/server.test.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
+const request = require('supertest');
+const server = require('./server')
+const db = require('../dbConfig')
+
+describe('Testing Server Connection and Setup', () => {
+    beforeAll(async () => {
```
Comment (juliankohlman) [inline api/server.test.js:6]: I know all this setup can be tedious but this is great practice and shows you're serious about the profession/craft 🔥

**Review by juliankohlman — COMMENTED** [review]
Sprint looks good Stefan 🔥 The testing module got a little bumpy for the section as a whole, but I think you handled it well and submitted some very thorough and readable code that meets all the spec's requirements 👍
